website/docs/language/settings/backends/s3.mdx

```diff
@@ -18,6 +18,12 @@ It is highly recommended that you enable
 on the S3 bucket to allow for state recovery in the case of accidental deletions and human error.
 :::
 
+## Bootstrap the backend using Amazon CloudFormation
+
+Amazon provides a sample repository for bootstrapping the backend with Amazon CloudFormation, accessible at the following link: [bootstrap-amazon-s3-remote-backend-for-open-tofu
+](https://github.com/aws-samples/bootstrap-amazon-s3-remote-backend-for-open-tofu).
```

Comment:
This repo is gone (404)

Reply:
Did you copy the `)`? It works for me.

Reply:
Weird, it works now. 
I did search for tofu in the `aws-samples` org with no results, but I see it now.